cvat-ui/src/index.tsx

```diff
@@ -30,19 +31,23 @@ interface StateToProps {
     userInitialized: boolean;
     usersInitialized: boolean;
     usersFetching: boolean;
+    aboutInitialized: boolean;
+    aboutFetching: boolean;
     formatsInitialized: boolean;
     formatsFetching: boolean;
     installedAutoAnnotation: boolean;
     installedTFSegmentation: boolean;
     installedTFAnnotation: boolean;
     notifications: NotificationsState;
     user: any;
+    about: any;
```

Comment:
Do you actually need this prop?

Reply:
No, this was something I was planning for before. My bad I didn't remove it.